src/types/metadata.ts

```diff
@@ -99,22 +100,23 @@ export interface StrategiesMetadata {
 
 export interface VaultMetadataOverrides {
```

Comment:
Sorry about the hard to review refactor, sorted these alphabetically for easier reading